core/src/avm2.rs

```diff
@@ -95,13 +101,60 @@ pub struct Avm2<'gc> {
     pub debug_output: bool,
 }
 
+pub struct ValueStackFrame<'a, 'gc> {
```

Comment:
I might be tempted to call this `OperandStack`... although I guess it doesn't hold `Value`s. Not sure :-)